<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSlider" name="hSlider">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>341</width>
     <height>18</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="list">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>40</y>
     <width>100</width>
     <height>421</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="myform" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>19</y>
     <width>471</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pause">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>420</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_next">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>420</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_speed">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>420</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_before">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>420</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_stop">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>420</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_add">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>380</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>V+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sub">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>380</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>V-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_choose">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>420</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_list">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>默认列表</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
